From b0f7ca30c5f8d4b5f639a4a5f7ee07caf2345696 Mon Sep 17 00:00:00 2001
From: Jocelyn Falempe <jfalempe@redhat.com>
Date: Fri, 20 Mar 2026 10:22:42 +0100
Subject: [PATCH] BACKPORT: gdm.service.in: Also conflict with kmsconvt

kmsconvt is a replacement for getty, so it will also conflict with gdm

Signed-off-by: Jocelyn Falempe <jfalempe@redhat.com>
Part-of: <https://gitlab.gnome.org/GNOME/gdm/-/merge_requests/357>
(cherry picked from commit f0c522d9df0c1c34bfccbb1f0eb2a8298c70fc20)
Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 data/gdm.service.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/gdm.service.in b/data/gdm.service.in
index 17e8a8de8..398aafff4 100644
--- a/data/gdm.service.in
+++ b/data/gdm.service.in
@@ -1,9 +1,9 @@
 [Unit]
 Description=GNOME Display Manager
 
-# replaces the getty
-Conflicts=getty@tty${GDM_INITIAL_VT}.service
-After=getty@tty${GDM_INITIAL_VT}.service
+# replaces the getty/kmsconvt
+Conflicts=getty@tty${GDM_INITIAL_VT}.service kmsconvt@tty${GDM_INITIAL_VT}.service
+After=getty@tty${GDM_INITIAL_VT}.service kmsconvt@tty${GDM_INITIAL_VT}.service
 
 # replaces plymouth-quit since it quits plymouth on its own
 Conflicts=${PLYMOUTH_QUIT_SERVICE}
-- 
2.53.0

